packages/amplify_auth_cognito/example/ios/unit_tests/MockErrorConstants.swift

```diff
@@ -16,6 +16,8 @@
 import Foundation
 import Amplify
 
+let MockErrorTemplate: String = "The operation couldn’t be completed. (AmplifyPlugins.AWSCognitoAuthError error"
```

Comment:
This isn't the right expectation though right? This seems like we failed to serialize.